planemo/galaxy/test/actions.py

```diff
@@ -131,6 +131,19 @@ def handle_reports_and_summary(ctx, structured_data, exit_code=None, kwds={}):
     return exit_code if exit_code is not None else summary_exit_code
 
 
+def merge_reports(input_paths, output_path):
+    reports = []
+    for path in input_paths:
+        with io.open(path, encoding='utf-8') as f:
+            reports.append(json.load(f))
+    tests = []
+    for report in reports:
+        tests.extend(report["tests"])
+    merged_report = {"tests": tests for report in reports}
```

Comment:
```suggestion
    merged_report = {"tests": tests}
````

Reply:
Good one ... what was I thinking ?

Reply:
Looks like you wanted to do the short way and then decided to do the more readable way :)